staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -226,12 +226,22 @@ else {
 }
 
 if ($enableAll -or $enableSnapshotSize) {
-  Write-Host "Collecting container snapshot size using DU tool"
-  if (Test-Path "C:\aks-tools\DU\du.exe") {
-    C:\aks-tools\DU\du.exe /accepteula
-    C:\aks-tools\DU\du.exe -l 1 C:\ProgramData\containerd\root\io.containerd.snapshotter.v1.windows\snapshots\ > "$ENV:TEMP\$timeStamp-du-snapshot-folder-size.txt"
-    $paths += "$ENV:TEMP\$timeStamp-du-snapshot-folder-size.txt"
+  Write-Host "Collecting actual size of snapshot (without sparse file sizes included)"
```

Comment:
@junjiezhang1997 can we set `-enableSnapshotSize` in one of AgentBaker Windows E2E to cover these lines?

Reply:
@junjiezhang1997 any thoughts?

Reply:
@kiashok how long does this step take? It may timeout in test with run-command if it takes too long.

Reply:
Hi @AbelHu , it could take long depening on number and size of the snapshots. That is why I have added this command under the "enableAll" or "enableSnapshotSize" flag so that it is not collected by default. Are there any concerns with this?